docs/cgmanifest.json

```diff
@@ -44,6 +44,15 @@
                     "commitHash": "59661c3f883dfd39cef6dc8eaf2fcbaae53597e8"
                 }
             }
+        },
+        {
+            "component": {
+                "type": "git",
+                "git": {
+                    "repositoryUrl": "https://github.com/google/benchmark",

```

Comment:
Similarly, if we're using `google/benchmark` purely as a submodule, my understanding is that we shouldn't update `cgmanifest.json`. This is for when we've copied code into the `microsoft/STL` repo proper (or substantially derived code). For example, we never listed `vcpkg` here when we used it as a submodule.

`llvm-project`'s presence here is confusing because it exists as a submodule *and* we have derived code from it: https://github.com/microsoft/STL/blob/17fde2cbab6e8724d81c9555237c9a623d7fb954/tests/std/tests/P0088R3_variant/test.cpp#L7
(Technically, the `llvm-project` hash listed here only needs to be the version that we picked up for derived code, but when we last updated the submodule, we also updated this hash to match - it is okay for the hash to be newer.)